### ISSUE TITLE
What are the steps to install LGSVL? 

### ISSUE THREAD

**Issue by AHPUymhd** [MayDGT/LeGEND#2]
![image](https://github.com/user-attachments/assets/41b6a474-bf18-4475-a880-cf04ce7f9360)
An error is reported when following this link to install it：
![image](https://github.com/user-attachments/assets/551b7f97-1de0-423b-a586-5f1b642f8793)
Excuse me, are you also following the steps above this link to install LSV?
Could you please tell us about the specific installation steps?Thank you so much！
@MayDGT

**Comment by MayDGT** [comment]
> ![image](https://private-user-images.githubusercontent.com/72792297/383382926-41b6a474-bf18-4475-a880-cf04ce7f9360.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA4NjE3NTQsIm5iZiI6MTczMDg2MTQ1NCwicGF0aCI6Ii83Mjc5MjI5Ny8zODMzODI5MjYtNDFiNmE0NzQtYmYxOC00NDc1LWE4ODAtY2YwNGNlN2Y5MzYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA2VDAyNTA1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwZDViZjY5ZGZiZTY5YTMyNTZiMTc5ODdkOTkxMmEyNmEyYTAxZTUzNmEwZDU2NmUyNWZlZTcyY2JjODdkYjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.fbTi6HMPyw-TlkRm8RxE8I_RuWeDB3KAljpEqSToBUI) An error is reported when following this link to install it： ![image](https://private-user-images.githubusercontent.com/72792297/383383048-551b7f97-1de0-423b-a586-5f1b642f8793.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA4NjE3NTQsIm5iZiI6MTczMDg2MTQ1NCwicGF0aCI6Ii83Mjc5MjI5Ny8zODMzODMwNDgtNTUxYjdmOTctMWRlMC00MjNiLWE1ODYtNWYxYjY0MmY4NzkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA2VDAyNTA1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1MzI4OGQ5MTQ3MWU0MTg4MDBjYTdmODkzMmI4MmEyMjg3ZGJmMjAwYTVjNTY2OTgzNmY3NGZjNzBjY2RiZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9lAAOToekz6DUh_yIgGclwjR-k__fTKAJO9ZMpROYns) Excuse me, are you also following the steps above this link to install LSV? Could you please tell us about the specific installation steps?Thank you so much！ @MayDGT

Yes, I installed LGSVL by following the steps in the provided link. I haven't encountered this error before. Perhaps you could check your network connection and try again.

**Comment by AHPUymhd** [comment]
![image](https://github.com/user-attachments/assets/1ae24f12-5bb6-424b-be38-8ef758b7eeec)
So do you start installing from Built With or from To Run the Project?
Thank you！
@MayDGT

**Comment by MayDGT** [comment]
> ![image](https://private-user-images.githubusercontent.com/72792297/383409379-1ae24f12-5bb6-424b-be38-8ef758b7eeec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyNDE5MTQsIm5iZiI6MTczMjI0MTYxNCwicGF0aCI6Ii83Mjc5MjI5Ny8zODM0MDkzNzktMWFlMjRmMTItNWJiNi00MjRiLWJlMzgtOGVmNzU4YjdlZWVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDAyMTMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwNjkwNDk3ZjU4ZDA3N2EzZjZhNWU2ZWU4NGMzZWY5MDUxYmM0NzEwZmYwNjNlZmMzMTFmMjRlNDg1MmJhZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Bu8EBQVMJfFQSP_w5RLqmmTV7tie_73GKfDT6pDba24) So do you start installing from Built With or from To Run the Project? Thank you！ @MayDGT

Sorry for the late reply. I started installing from **To Run the Project**.
